migrations/227_grading_method_split.sql

```diff
@@ -0,0 +1,33 @@
+-- Add new grading columns
+ALTER TABLE submissions 
+    ADD COLUMN IF NOT EXISTS grading_method_internal boolean DEFAULT True,
+    ADD COLUMN IF NOT EXISTS grading_method_external boolean DEFAULT False,
+    ADD COLUMN IF NOT EXISTS grading_method_manual boolean DEFAULT False;
+ALTER TABLE questions
+    ADD COLUMN IF NOT EXISTS grading_method_internal boolean DEFAULT True,
+    ADD COLUMN IF NOT EXISTS grading_method_external boolean DEFAULT False,
+    ADD COLUMN IF NOT EXISTS grading_method_manual boolean DEFAULT False;
+ALTER TABLE grading_jobs
+    ADD COLUMN IF NOT EXISTS grading_method_internal boolean DEFAULT True,
+    ADD COLUMN IF NOT EXISTS grading_method_external boolean DEFAULT False,
+    ADD COLUMN IF NOT EXISTS grading_method_manual boolean DEFAULT False;
+
+-- Update based on old values
+UPDATE submissions 
```

Comment:
Here's the update that may break on an already migrated deploy. I can add a column check on the table if needbe (i.e. only run these `UPDATE` statements if the `grading_method` column is still hanging around.